earth_enterprise/src/common/config/geConfigUtil.cpp

```diff
@@ -42,15 +44,28 @@ bool IsRedHat(void) {
 
 uint GetNumCPUs(void) {
   uint numcpus = 0;
-  std::ifstream in("/proc/cpuinfo");
+  /*std::ifstream in("/proc/cpuinfo");
```

Comment:
I think we should write in C++ instead of pretending that C++ is BASH.

In C++11 you can use http://en.cppreference.com/w/cpp/thread/thread/hardware_concurrency.

Otherwise, POSIX defines `sysconf`(3) for that purpose, e.g.:

```C++
#include <unistd.h>

long cpu_count = sysconf(_SC_NPROCESSORS_ONLN);
```

Reply:
It might be better to use sysconf until we are actually using 11 (it is currently still in review??)

Either way, this function must change, as well as GetPhysicalMemorySize( ) below because, as they stand, they are nothing more than re-interpretations of `grep | wc -l`